packages/swingset-liveslots/src/vatDataTypes.ts

```diff
@@ -171,17 +178,21 @@ export type VatData = {
   /** @deprecated Use defineVirtualExoClass instead */
   defineKind: <P, S, F>(
     tag: string,
+    // @ts-expect-error rest parameter must be of an array type
     init: (...args: P) => S,
```

Comment:
must be array or `Parameters` type, right? I suppose there's no way to say "<P extends Parameters, ...>"?

I'm a little surprised that this works; i.e. that the types of the init parameters get checked. Do we happen to have a type test?

Reply:
I didn't invest in fixing these because they're all on `@deprecated` types